nav2_map_server/include/nav2_map_server/map_saver.hpp

```diff
@@ -23,23 +23,53 @@
 namespace nav2_map_server
 {
 
+#define FREE_THRESH_DEFAULT 25
```

Comment:
These should be parameterized not globally set

Reply:
Fixed in https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-e22d88faf6e8ac7a5338c8c6cbc2f7a4R76, https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-99885e6562ca50edd8cc80db6362d317R46